methods.tex

```diff
@@ -146,6 +145,7 @@ \subsection*{Funding}
 accelerated linear algebra.
 
 Finally, since May 2019 the project receives around 10K USD annually from
+% typically is correct right now, but do we want to write it so specifically?
```

Comment:
Was just curious if we want to keep it so specific (it is fully correct now of course). I am happy with it as is though.

Reply:
"a small amount" may suffice; it's a small enough amount that it probably doesn't matter either way :)

Reply:
Let's go with small amount.